### PR TITLE
Fabo/develop preview

### PR DESCRIPTION
**Pull request by faboweb**
Add develop preview and adds nice urls to previews

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5ef382335ab7e8e550f7fefa--lunieio.netlify.app

File: .github/workflows/app_deploy_develop.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Publish on Netlify
+on:
+  push:
+    branches:
+      - "develop"
```
Comment (Bitcoinera) [inline .github/workflows/app_deploy_develop.yml:5]: Nice! But hey, how comes it was working before? I'm confused 👀

Comment (faboweb) [reply .github/workflows/app_deploy_develop.yml:5]: I disabled the Netlify automatic deployment so we only build and deploy the things we want to reduce costs with Netlify

Comment (Bitcoinera) [reply .github/workflows/app_deploy_develop.yml:5]: oh ok ok. Nice!

**Pull request merged**